modules/util/portal-tools-bundle-support/src/main/java/com/liferay/portal/tools/bundle/support/commands/InitBundleCommand.java

```diff
@@ -43,6 +46,9 @@
 )
 public class InitBundleCommand extends DownloadCommand {
 
+	public InitBundleCommand() {
+	}
```

Comment:
I think this constructor isn't needed.  Removing it will help simplify the code.